docs/interactions/Application_Commands.md

```diff
@@ -1225,8 +1225,8 @@ Takes a list of application commands, overwriting the existing command list for
 | default_member_permissions? | ?string                                                                                                                                        | Set of [permissions](#DOCS_TOPICS_PERMISSIONS) represented as a bit set                                                                                                                 |
 | dm_permission?              | ?boolean                                                                                                                                       | **Deprecated (use `contexts` instead)**; Indicates whether the command is available in DMs with the app, only for globally-scoped commands. By default, commands are visible.           |
 | default_permission?         | boolean                                                                                                                                        | Replaced by `default_member_permissions` and will be deprecated in the future. Indicates whether the command is enabled by default when the app is added to a guild. Defaults to `true` |
-| integration_types          | list of [integration types](#DOCS_RESOURCES_APPLICATION/application-object-application-integration-types)                                      | [Installation context(s)](#DOCS_RESOURCES_APPLICATION/installation-context) where the command is available, defaults to `GUILD_INSTALL` (`[0]`)                                                         | all         |
-| contexts                   | list of [interaction context types](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object-interaction-context-types)                  | [Interaction context(s)](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object-interaction-context-types) where the command can be used, defaults to all contexts `[0,1,2]`                   | all         |
+| integration_types?         | list of [integration types](#DOCS_RESOURCES_APPLICATION/application-object-application-integration-types)                                      | [Installation context(s)](#DOCS_RESOURCES_APPLICATION/installation-context) where the command is available, defaults to `GUILD_INSTALL` (`[0]`)                                                         | all         |
+| contexts?                  | list of [interaction context types](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object-interaction-context-types)                  | [Interaction context(s)](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object-interaction-context-types) where the command can be used, defaults to all contexts `[0,1,2]`                   | all         |
```

Comment:
The last row looks wrong
Tables my beloved

Reply:
really? i preserved the spacing from before

Reply:
Resolved by https://github.com/discord/discord-api-docs/commit/2d265c53897b49cc0fdc059f0c7da8a178a80f74